components/nav/MobileNav.tsx

```diff
@@ -35,6 +35,7 @@ const MobileNav = () => {
                         alt="Barrese Bookkeeping"
                         onClick={() => {
                             router.push('/')
+                            setShow(false)
```

Comment:
Best to do the state setting first, then change the route. This could lead to potential memory leaks. 

Also, dear god. Take this out of the `onClick` and into it's own function that the `onClick` calls 🤣